lua/guard-collection/init.lua

```diff
@@ -0,0 +1 @@
+return true
```

Comment:
hmm what does this line do ?

Reply:
So `pcall(require, "guard-collection")` tells one if it's available, or one would have to require guard-collection.formatter to do so

Reply:
use `nvim_get_runtime_file`